### PR TITLE
EZP-29737: Invalid UDW container height after resizing window

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/EZP-29737

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

File: src/modules/universal-discovery/universal.discovery.module.js [hunk 1]
```diff
@@ -989,6 +1027,14 @@ export default class UniversalDiscoveryModule extends Component {
         return <span className="m-ud__location-not-allowed">{locationNotAllowedMessage}</span>;
     }
 
+    setMainContainerRef(ref) {
+        this._refMainContainer = ref;
```
Comment (tischsoic) [inline src/modules/universal-discovery/universal.discovery.module.js]: I would go for `React.createRef()`.

Comment (tischsoic) [inline src/modules/universal-discovery/universal.discovery.module.js:1026]: I would go for `React.createRef()`.

Comment (tischsoic) [reply src/modules/universal-discovery/universal.discovery.module.js:1026]: But if we want to stick with this way in order to look it the same as in other places, then OK.

Comment (sunpietro) [reply src/modules/universal-discovery/universal.discovery.module.js:1026]: I wanted to keep the consistency. If we decide to use `React.createRef()` here and there, then we should change it everywhere in this repository.

**Review by tischsoic — APPROVED** [review]

**Comment by sunpietro** [comment]
@micszo can you take care of it, or assign someone, please? :)

**Comment by micszo** [comment]
@sunpietro ok :) could you buff up the description with some steps?

**Comment by sunpietro** [comment]
@micszo and other QA members: change the browser window height from tall to short and you should notice, that with this fix the buttons at the bottom of UDW don't disappear below the fold anymore.

**Review by barbaragr — CHANGES_REQUESTED** [review]
When I change height, buttons slide down outside the UDW
[udw_btns.zip](https://github.com/ezsystems/ezplatform-admin-ui-modules/files/2491551/udw_btns.zip)

**Comment by sunpietro** [comment]
@barbaragr this is expected behaviour, because the container height is updated with a delay, to avoid unneccesary calculations which are really expensive in terms of performance.

**Review by barbaragr — APPROVED** [review]

**Comment by lserwatka** [comment]
@sunpietro you can merge it up

**Comment by sunpietro** [comment]
@lserwatka done :)

**Pull request merged**